Presentation.Web/Tests/Helpers/InterfaceHelper.ts

```diff
@@ -1,63 +1,66 @@
 ﻿import CSSLocator = require("../object-wrappers/CSSLocatorHelper");
 import Select2 = require("./Select2Helper");
 import constant = require("../Utility/Constants")
+import interfaceCatalogHelper = require("./InterfaceCatalogHelper");
 
 class InterfaceHelper {
-    private const = new constant();s
-    private cssHelper = new CSSLocator();
+    private static const = new constant();
+    private static cssHelper = new CSSLocator();
 
-    public writeDataToAllInputs(data: string, exposedBy: string, sysInterface: string, access: string, belongsTo: string, dataTypeTable: string) {
-        return this.writeDataToTextInput(data, this.const.interfaceNameInput)
-            .then(() => {
-                return this.writeDataToTextInput(data, this.const.interfaceIdInput);
-            }).then(() => {
-                return this.writeDataToTextInput(data, this.const.interfaceVersionInput);
-            }).then(() => {
-                return this.writeDataToTextInput(data, this.const.interfaceDescriptionInput);
-            }).then(() => {
-                return this.writeDataToTextInput(data, this.const.interfaceDescriptionLinkInput);
-            }).then(() => {
-                return this.writeDataToTextInput(data, this.const.interfaceNoteInput);
-            }).then(() => {
-                return this.selectDataFromSelect2Field(exposedBy, this.const.interfaceSelectExhibit);
-            }).then(() => {
-                return this.selectDataFromSelect2Field(sysInterface, this.const.interfaceSelectInterface);
-            }).then(() => {
-                return this.selectDataFromSelect2Field(access, this.const.interfaceSelectAccess);
-            }).then(() => {
-                return this.selectDataFromSelect2Field(belongsTo, this.const.interfaceSelectBelongs);
-            }).then(() => {
-                return this.writeDataToTable(data, dataTypeTable);
-            });
+    public static writeDataToAllInputs(nameOfInterface: string, data: string, exposedBy: string, sysInterface: string, access: string, belongsTo: string, dataTypeTable: string, version: string) {
+        return interfaceCatalogHelper.gotoSpecificInterface(nameOfInterface).then(() =>
+            this.writeDataToTextInput(data, this.const.interfaceNameInput)
+                .then(() => {
+                    return this.writeDataToTextInput(data + "1", this.const.interfaceIdInput);
```

Comment:
@JacobMalling Hvorfor ikke være eksplicit med de der felter.. ja der kommer mange parametre, men det er jo sandheden. I stedet afhænger verifyDataSaved af at denne har været kaldt først, for den implementerer samme algoritme.